argo_rollouts/enable-support-for-namespace-scoped-argo-rollouts-installation.adoc

```diff
@@ -0,0 +1,27 @@
+:_mod-docs-content-type: ASSEMBLY
+include::_attributes/common-attributes.adoc[]
+[id="enable-support-for-namespace-scoped-argo-rollouts-installation"]
+= Enable support for namespace-scoped Argo Rollouts installation
+:context: enable-support-for-namespace-scoped-argo-rollouts-installation
+
+toc::[]
+
+By default, the {gitops-title} installation enables support for the cluster-scoped Argo Rollouts installation. In this mode of installation, the Argo Rollouts custom resources (CRs) defined in any namespace are reconciled by that Argo Rollouts instance. This allows you to use Argo Rollouts CR across any namespace on the cluster.
+
+The namespace-scoped Argo Rollouts installation is an alternative to the cluster-scoped Argo Rollouts installation. This mode of installation does not require cluster-wide permissions by using `cluster roles` or `cluster role binding`. This means that only users with permissions that are limited to a specific namespace can use such resources.
+
+A namespaced-scoped instance provides security benefits by limiting the cluster scope of a single Argo Rollouts instance. With this mode of installation, Argo Rollouts only handles Argo Rollouts CR within the same namespace.
+
+For example, if Argo Rollouts is installed in the 'argo-rollouts-ns' namespace, the Argo Rollouts installation only handles Argo Rollouts CR in the 'argo-rollouts-ns' namespace. The Argo Rollouts CR installed in another namespace, for example, the 'my-application' namespace, are not used.
```

Comment:
How about this, making it slightly more clear that this paragraph continues to refer to namespace-scoped instances (I want to make sure that people who are only skimming are not confused):

For example, if **namespace-scoped** Argo Rollouts is installed in the 'argo-rollouts-ns' namespace, **that** Argo Rollouts installation only handles Argo Rollouts CR in the 'argo-rollouts-ns' namespace. **An** Argo Rollouts CR installed in another namespace, for example, the 'my-application' namespace, **is** not **handled**.

(changes are bolded)

Reply:
This feedback has been reworded as per the reviewer's comment. Fixed.